org.mango3d.LycheeSlicer.yml

```diff
@@ -0,0 +1,54 @@
+app-id: org.mango3d.LycheeSlicer
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: lycheeslicer
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --filesystem=home
```

Comment:
Why do you need this?

Reply:
filesystem=home is so stl files can be imported from the menu
device=dri is so the 3d runs smoothly
share=network is because it neets network to login

Reply:
> filesystem=home is so stl files can be imported from the menu

Does that go through a filechooser? Does it support filechooser portals? Then you don't need this.

Reply:
it uses electron and it has a gtk3 file picker. When I tried to run it without the file permissions no files showed up, even with GTK_USE_PORTAL=1. How would I let it use the FileChooser portals?

Reply:
What version of Electron is that? It is supposed to use the filechooser portal since electron 14.

If you run `/usr/libexec/xdg-desktop-portal -vr` and open the filechooser, it should say "handling openfile" if it's using the portal.

Reply:
i ran `strings lycheeslicer | grep 'Electron/[0-9]' ` and it returned Electron/12.2.3, and xdg-desktop-portal didn't show anything when i opened the filechooser

Reply:
Yea that's too old an EOL/insecure version of Electron